packages/react/src/use-action-form/utils.ts

```diff
@@ -357,7 +357,9 @@ export const reshapeDataForGraphqlApi = async (client: AnyClient, defaultValues:
           return getParentRelationshipFieldGraphqlApiInput({ input, result });
         case "BelongsTo":
           return inputHasId
-            ? inputHasMoreFields
+            ? input["id"] === null
+              ? { _link: null }
+              : inputHasMoreFields
```

Comment:
The only difference I can imagine here is that something like 
`belongsToField: {id:null, otherField:"non-null"}` 
will now unlink the record instead of trying to update a record without an ID. This feels more correct. 

Since the other inputs are ignored when `input["id"] === null && inputHasMoreFields`, should there be  an error/warning to indicate that?

Reply:
>trying to update a record without an ID

How would you update a record without an ID?

Reply:
>> trying to update a record without an ID

> How would you update a record without an ID?

I mean that you are no longer able to attempt an update with `id:null`. It would be a GQL error anyways, even with unique fields on the related